index.bs

```diff
@@ -306,14 +306,13 @@ A <dfn>registry entry</dfn> is a record with the following fields:
 1. If _entry_.[[State]] is "ready", return a new error promise.
 1. If _entry_.[[Instantiate]] is not *undefined*, return _entry_.[[Instantiate]].
 1. Let _hook_ be _loader_.[[Instantiate]].
-1. Let _p_ be the result of transforming RequestFetch(_loader_, _key_) with a fulfillment handler that, when called with argument _payload_, runs the following steps:
-  1. Return the result of transforming RequestTranslate(_loader_, _key_) with a fulfillment handler that, when called with argument _source_, runs the following steps:
-    1. // TODO: metadata
-    1. Let _p1_ be the result of promise-calling _hook_(_key_, _payload_, _source_).
```

Comment:
Looking at this again - I guess that the `_payload_` comes from fetch and the `_source_` from translate which would explain this signature. So it looks like the instantiate hook is called with arguments - `key`, `originalSource`, `translatedSource`? May be worth moving this out for further clarification for now.